cpp/src/holtwinters/hw_math.cu

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "hw_cu_utils.hpp"
+#include "hw_math.hpp"
```

Comment:
Just use `hw_math.h` and pull all impls into there. This is only used internally by the HoltWinters impl and seems unlikely we would need to swap out impls in the future.